features/old/buyers/users.feature

```diff
@@ -120,7 +120,7 @@ Feature: Buyer users management
     Then I should see "User was successfully updated."
 
   Scenario: Fields edition by provider
-    Given provider "foo.3scale.localhost" has the following fields defined for "User":
+    Given provider "foo.3scale.localhost" has the following fields defined for users:
```

Comment:
related to previous comment, this sounds better than the `users have the following custom fields:`